.ci/copy-site-to-sourceforge.sh

```diff
@@ -16,7 +16,7 @@ fi
 checkForVariable "SF_USER"
 
 echo "Creating shell for $SF_USER@shell.sourceforge.net"
-ssh -i ~/.ssh/private_sourceforge_key -t "$SF_USER"@shell.sourceforge.net create
+ssh -i ~/.ssh/private_sourceforge_key "$SF_USER"@shell.sourceforge.net create
```

Comment:
This resolves
> during execution on local it does demand from me password in interactive mode. .... somethig is missed on my local setup
after shh creation phase it waited for "exit" to continue shell execution

Got this tip from sourceforge when I tried to connect
```
If you tell ssh to allocate a tty (e.g. using -t), an interactive shell will be opened
when the create is done. Otherwise, the create command will exit when the shell
becomes ready for use.
```

We do not want to use this shell interactively so the `-t` shell must be removed.